src/modules/projects/components/tables/ProjectExternalSubmissionsTable.tsx

```diff
@@ -161,6 +162,7 @@ const ProjectExternalSubmissionsTable = ({
               Bulk Review ({selectedRows.length}) Submissions
             </LoadingButton>
           ),
+          additionalActions: <RefreshExternalSubmissionsButton />,
```

Comment:
Sort of a nit, but I don't think it needs to be visible when the bulk action button is present. The UI feels busy and it's unclear if refresh interacts with bulk somehow.

We could display it as part of the title instead:
```
          title: (
            <Stack
              direction='row'
              justifyContent={'space-between'}
              sx={{ width: '100%' }}
            >
              <Typography variant='h3'>Form Submissions</Typography>
              <RefreshExternalSubmissionsButton />
            </Stack>
          ),
```